src/utils/MasterFisher.ts

```diff
@@ -12,6 +12,19 @@ export class MasterFisher implements utils.Fishable {
     public external = new fhirdefs.FHIRDefinitions()
   ) {}
 
+  fishForStructureDefinition(item: string) {
+    const json = this.fishForFHIR(
+      item,
+      utils.Type.Resource,
+      utils.Type.Profile,
+      utils.Type.Extension,
+      utils.Type.Type
+    );
+    if (json) {
+      return fhirtypes.StructureDefinition.fromJSON(json);
```

Comment:
Good catch, we definitely don't want to fail here. Since the function this happens in is `fishForStructureDefinition`, and `StructureDefinition` class requires a snapshot, I've updated it so that if we cannot create an instance of our `StructureDefinition` class, this function just returns `undefined`. So it acts as if the SD does not exist at all.